.circleci/run_docker_build.sh

```diff
@@ -3,7 +3,7 @@
 # NOTE: This script has been adapted from content generated by github.com/conda-forge/conda-smithy
 
 REPO_ROOT=$(cd "$(dirname "$0")/.."; pwd;)
-IMAGE_NAME="condaforge/linux-anvil-comp7"
+IMAGE_NAME="condaforge/linux-anvil-cuda:10.0"
```

Comment:
So this is fine for testing purposes, but we will need to revert this before merging.

Reply:
Fixed!

Reply:
You need to keep this, remove skipping, make tests pass. Once that is reviewed, we can revert this change and merge.

Reply:
Ok!

Reply:
Since you are using 10.0 image, you should skip other versions in staged-recipes and only enable them in the feedstock.